docs/guides/stonesense-art-guide.rst

```diff
@@ -0,0 +1,64 @@
+.. _stonesense-art-guide:
+
+Stonesense art creation guide
+=============================
+
+Understanding sprites
+---------------------
+
+Understanding how Stonesense deals with sprites is central to anyone who wishes to modify the content. The
+scheme is not very complicated, and this guide will give a short introduction to how they work. With the
+exception of floors, which we will discuss later, all sprites are 32x32 pixels big and come in groups known
+as Sprite Sheets. All sprites are loaded and rendered in 32-bit full-color PNGs. The image files should have
+a transparent background but pure magenta (RGB: 255,0,255) is also treated as transparent.
```

Comment:
I think this section needs an introduction to isometric perspective. otherwise the following examples (especially the "template for the area" example) are hard to follow